src/main/java/org/gridsuite/directory/server/DirectoryController.java

```diff
@@ -111,14 +112,14 @@ public DirectoryController(DirectoryService service) {
         return ResponseEntity.ok().body(service.deleteElement(elementUuid, userId));
     }
 
-    @RequestMapping(method = RequestMethod.HEAD, value = "/directories/{directoryUuid}/{elementName}")
-    @Operation(summary = "Check if the element exists")
+    @RequestMapping(method = RequestMethod.HEAD, value = "/directories/{directoryUuid}/{elementName}/{type}")
+    @Operation(summary = "Check if the element exists in the given directory")
```

Comment:
`Check if an element with this name and this type already exists in the given directory `?